### PR TITLE
ETCD-662: fix: etcd-backup-server prune path

### PR DESCRIPTION
**Pull request by Elbehery**
follow up on https://github.com/openshift/cluster-etcd-operator/pull/1340

### CODE REVIEW

File: pkg/cmd/backuprestore/backupserver.go [hunk 1]
```diff
@@ -34,7 +34,6 @@ func (b backupRunnerImpl) runBackup(backupOpts *backupOptions, pruneOpts *prune.
 		return err
 	}
 
-	pruneOpts.BackupPath = backupVolume + "/" + dateString
```
Comment (tjungblu) [inline pkg/cmd/backuprestore/backupserver.go:37]: now that I'm reading this again, `paths.Join` would also be a safer option

Comment (Elbehery) [reply pkg/cmd/backuprestore/backupserver.go:37]: yes u r right, should have written test :'( 

fixing ...

Comment (Elbehery) [reply pkg/cmd/backuprestore/backupserver.go:37]: updated 👍🏽

**Pull request merged**